airflow/config_templates/config.yml

```diff
@@ -492,6 +492,14 @@ core:
       type: string
       example: ~
       default: "Disabled"
+    max_templated_field_size:
+      description: |
+        The maximum size of the rendered template field in bytes. If the value to be stored in the
+        rendered template field exceeds this size, it's redacted.
+      version_added: 2.9.0
+      type: integer
+      example: ~
+      default: "1024"
```

Comment:
NIT. 1024 looks like rather low default value. I'd put it to 4096 (another very opinionated magic value here) but also I'd mention that as significant newsfragment as it might make big difference for some people..